lasagne/init.py

```diff
@@ -132,6 +279,16 @@ def sample(self, shape):
 
 
 class Sparse(Initializer):
+    """Initialize weights as sparse matrix.
+
+    Parameters
+    ----------
+    sparsity : float
+        Fraction of weights with non-zero value. Larger value gives less
+        sparsity.
```

Comment:
Yes, either this, or my second suggestion, whatever you find easier to understand. I just think it's important to note that it's not just the fraction of the full matrix, but even per column, and that it's not stochastic, but counted exactly.